week-3/slideshow/slideshow.js

```diff
@@ -1,8 +1,81 @@
 const images = [
-    "./assets/cute-cat-a.png",
-    "./assets/cute-cat-b.jpg",
-    "./assets/cute-cat-c.jpg",
+  "./assets/cute-cat-a.png",
+  "./assets/cute-cat-b.jpg",
+  "./assets/cute-cat-c.jpg",
+  "./assets/cute-cat-d.jpg",
+  "./assets/cute-cat-e.webp",
+  "./assets/cute-cat-f.jpg",
+  "./assets/cute-cat-g.jpg",
+  "./assets/cute-cat-h.webp",
+  "./assets/cute-cat-i.jpg",
+  "./assets/cute-cat-j.jpg",
+  "./assets/cute-cat-k.jpg",
+  "./assets/cute-cat-l.jpg",
+  "./assets/cute-cat-m.webp",
 ];
 
+// Write your code here
+const backward = document.getElementById("backward-btn");
+const forward = document.getElementById("forward-btn");
+const imageCarousel = document.getElementById("carousel-img");
+const autoBackward = document.getElementById("auto-back-btn");
+const autoForward = document.getElementById("auto-forward-btn");
+const stop = document.getElementById("stop-btn");
+const inputTime = document.getElementById("inputTime");
```

Comment:
What happens if I click the auto backward or auto forward button without adding any input time value?